aion/web/elm/src/Update.elm

```diff
@@ -269,6 +307,32 @@ update msg model =
                 else
                     categoryFormValidationErrorToast (model ! [])
 
+        CreateNewRoom ->
+            let
+                roomForm =
+                    model.panelData.roomForm
+
+                validationErrors =
+                    []
+
+                subjectIds =
+                    List.map (\( x, _ ) -> x) (Multiselect.getSelectedValues model.panelData.categoryMultiSelect)
```

Comment:
I guess we could get a better naming than x. The moment somebody wants to change the logic here, they'll have to log what the x is. Also, do we still use the name 'subject'?